### PR TITLE
Update Reverse-Proxy.md

### PR DESCRIPTION
**Pull request by ajnadox**
update Caddy installation

### CODE REVIEW

File: Reverse-Proxy.md [hunk 1]
```diff
@@ -121,12 +121,25 @@ Without SSL:
...
-
+Install Caddy server
+```bash
+sudo apt install -y debian-keyring debian-archive-keyring apt-transport-https -y
+curl -1sLf 'https://dl.cloudsmith.io/public/caddy/stable/gpg.key' | sudo gpg --dearmor -o /usr/share/keyrings/caddy-stable-archive-keyring.gpg
+curl -1sLf 'https://dl.cloudsmith.io/public/caddy/stable/debian.deb.txt' | sudo tee /etc/apt/sources.list.d/caddy-stable.list
+sudo apt-get update && sudo apt-get install caddy -y
+nano /etc/caddy/Caddyfile
+```
+edit nano /etc/caddy/Caddyfile, Delete everything in the file and replace it with (change to your domain):
```
Comment (CommanderStorm) [inline Reverse-Proxy.md:132]: See above

```suggestion
```

File: Reverse-Proxy.md [hunk 2]
```diff
@@ -121,12 +121,25 @@ Without SSL:
 ```
 
 # Caddy
-
+Install Caddy server
```
Comment (CommanderStorm) [inline Reverse-Proxy.md:124]: What is happening right here?
I think you are trying to install caddy here => linking their installation guide is the appropriate way to do that

We cannot maintain a way to install caddy in the wiki.

```suggestion
After [instally caddy](https://caddyserver.com/docs/install), edit your `Caddyfile` (by default at `/etc/caddy/Caddyfile`) and add the following:
```

File: Reverse-Proxy.md [hunk 3]
```diff
@@ -121,12 +121,25 @@ Without SSL:
...
+edit nano /etc/caddy/Caddyfile, Delete everything in the file and replace it with (change to your domain):
 ```nginx
 subdomain.domain.com {
     reverse_proxy 127.0.0.1:3001
 }
 ```
+Start and enable the Caddy service with:
+```bash
+sudo systemctl enable --now caddy
+```
```
Comment (CommanderStorm) [inline Reverse-Proxy.md:141]: Caddy supports more than just the linux instation instructions.
Please reflect this in the installation guide or link to the appropriate docs instead

File: Reverse-Proxy.md [hunk 4]
```diff
@@ -121,12 +121,25 @@ Without SSL:
...
+You should now be able to point a browser to https://subdomain.domain.com/ and see your Uptime Kuma site
```
Comment (CommanderStorm) [inline Reverse-Proxy.md:142]:
```suggestion
Uptime Kuma is now avaliable at https://subdomain.domain.com/
```

**Review by CommanderStorm — CHANGES_REQUESTED** [review]

**Comment by louislam** [comment]
Same reason as #80

**Pull request closed without merging**